### PR TITLE
#70 update user model and scripts and create new model Schedule

### PR DESCRIPTION
**Pull request by alesyalis**
create new model Schedule
add fild PrimarySkill into User
also add fild IsAssigned into ProjectResult
update all js scripts

### CODE REVIEW

File: Recrutify/Recrutify.Host/appsettings.Development.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "MongoSettings": {
-    "ConnectionString": "mongodb+srv://BestTeam2:123qwerty@recruitifyproject.epdnl.mongodb.net/RecruitifyTest"
+    "ConnectionString": "mongodb+srv://BestTeam2:123qwerty@recruitifyproject.epdnl.mongodb.net/RecruitifyDB"
```
[the remark this answers is not in the document]
Comment (alesyalis) [reply Recrutify/Recrutify.Host/appsettings.Development.json]: done

File: Recrutify/Recrutify.DataAccess/Models/UserPrimarySkill.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿using System;
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.DataAccess/Models/UserPrimarySkill.cs:1]: do you need this using?

File: Recrutify/Recrutify.Services/DTOs/ProjectResultDTO.cs [hunk 3]
```diff
@@ -11,6 +11,8 @@ public class ProjectResultDTO
 
         public StatusDTO Status { get; set; }
 
+        public bool IsAssign { get; set; }
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Services/DTOs/ProjectResultDTO.cs:14]: IsAssigned

File: Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Recrutify.DataAccess.Models
+{
+    public class ParticipantSlot
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs:6]: AssignedCandidate

File: Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        public IEnumerable<int> BestTimeToConnectCanditate { get; set; }
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs:8]: remove word Candidate from all namings of fields

File: Recrutify/Recrutify.DataAccess/Models/ScheduleSlot.cs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Recrutify.DataAccess.Models
+{
+    public class ScheduleSlot
+    {
+        public DateTime AvailableTime { get; set; }
+
+        public ParticipantSlot ParticipantSlot { get; set; }
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.DataAccess/Models/ScheduleSlot.cs:9]: AssignedCandidate, and don't forget to rename all fields in scripts too!

File: Recrutify/Recrutify.DataAccess/Models/UserPrimarySkill.cs (hunk 2 above)
Comment (alesyalis) [reply Recrutify/Recrutify.DataAccess/Models/UserPrimarySkill.cs:1]: done

File: Recrutify/Recrutify.Services/DTOs/ProjectResultDTO.cs (hunk 3 above)
Comment (alesyalis) [reply Recrutify/Recrutify.Services/DTOs/ProjectResultDTO.cs:14]: sorry, I did it right in the model but forgot to change it into DTO

File: Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs (hunk 4 above)
Comment (alesyalis) [reply Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs:6]: done

File: Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs (hunk 5 above)
Comment (alesyalis) [reply Recrutify/Recrutify.DataAccess/Models/ParticipantSlot.cs:8]: done

File: Recrutify/Recrutify.DataAccess/Models/ScheduleSlot.cs (hunk 6 above)
Comment (alesyalis) [reply Recrutify/Recrutify.DataAccess/Models/ScheduleSlot.cs:9]: done

**Review by elizavetapavliv — APPROVED** [review]

File: Recrutify/Recrutify.Host/DBScripts/candidates_collection.js [hunk 7]
```diff
@@ -282,19 +289,21 @@
...
                    "_id": UUID("a9962438-3775-11ec-8787-6b8ecd0a876a"),
                    "Name": "AutomationQA",
-               }
+               },
+               "IsAssigned": false
            }
        ],
        "CurrentJob": "PLC CosmosTV",
-       "AdditionalInfo": "Always available for calls"
+        "AdditionalInfo": "Always available for calls"
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/DBScripts/candidates_collection.js:306]: Extra space)

Comment (alesyalis) [reply Recrutify/Recrutify.Host/DBScripts/candidates_collection.js]: I'm surprised what you saw)

**Review by elizavetapavliv — APPROVED** [review]

**Pull request merged**